app/experimenter/experiments/models/nimbus.py

```diff
@@ -16,6 +16,20 @@
 from experimenter.projects.models import Project
 
 
+class NimbusExperimentManager(models.Manager):
+    def launch_queue(self, application):
+        return super().filter(
```

Comment:
You don't actually need to call super here, you can just call `self.filter` or `self.all().filter`.  You only need to call `super` if you're overriding the same method and need to invoke the parent implementation of it.